.ci/run.sh

```diff
@@ -61,4 +61,4 @@ sudo docker ps -a
 # Execute the tests under `clearcontainers/tests` repository.
 test_repo="github.com/clearcontainers/tests"
 cd "${GOPATH}/src/${test_repo}"
-sudo -E PATH=$PATH make check
+sudo -E PATH=$PATH TIMEOUT=15 make check
```

Comment:
@grahamwhaley it was only for testing purpose, that's why I put the do-not-merge label. But I raised another PR for that: https://github.com/clearcontainers/tests/pull/413